jobs/delorean/fuse-online/ga/discovery.yaml

```diff
@@ -27,6 +27,16 @@
           default: 'fuse-online'
           description: '[REQUIRED] Product to check, this affects the way the job verifies if a new version is available'
           read-only: true
+      - string:
+          name: 'templatesDir'
+          default: 'resources'
+          description: '[REQUIRED] directory where openshift templates are located'
+          read-only: true
+      - string:
+          name: 'registryProjectIDs'
+          default: 'fuse7, fuse7-tech-preview'
+          description: '[REQUIRED] registry image projects'
+          read-only: true
```

Comment:
Named consistently with rc equivalent :+1: https://github.com/integr8ly/ci-cd/blob/master/jobs/delorean/fuse-online/rc/discovery.yaml#L35-L44 Any reason we have the extra tech preview one here and not in the rc job?